### PR TITLE
Have mock-toolrunner emit stdline and errline events.

### PR DESCRIPTION
**Pull request by ILMTitan**
Fix for issue #227.

### CODE REVIEW

File: node/mock-answer.ts [hunk 1]
```diff
@@ -27,7 +27,11 @@ export class MockAnswers {
         this._answers = answers;
     }
 
-    public getResponse(cmd: string, key: string, debug: (message: string) => void): any {
+    public getResponse<T extends keyof TaskLibAnswers, K extends keyof TaskLibAnswers[T]>(
```
Comment (ericsciple) [inline node/mock-answer.ts]: does this work with typescript 1.8.7 compiler?

**Comment by ILMTitan** [comment]
Probably not. I believe it is a 2.0 feature. In a couple of days I will
change it back and just cast where I need to.

**Comment by ILMTitan** [comment]
Removed keyof and indexer types..

**Comment by ILMTitan** [comment]
@ericsciple Please take another look

**Comment by bryanmacfarlane** [comment]
I think it looks good.  Did you validate by writing tests on your end?

**Comment by hashtagchris** [comment]
Hi @ILMTitan, this looks like a great change. Do you think you'll be able to do some validation and finish up this PR?

**Comment by damccorm** [comment]
@ILMTitan any update on this?

**Comment by damccorm** [comment]
Closing this due to inactivity. This issue will be fixed with #462 which was based on this PR.

**Pull request closed without merging**